packages/cfd/src/sass/cfd-dashboard.scss

```
@@ -189,7 +189,9 @@
                 align-items: center;
                 justify-content: center;
                 flex-wrap: wrap;
-
+                &--column {
+                    flex-direction: column;
+                }
```

please add a space between selectors:
```suggestion
                }
                
```